eng/common/scripts/Invoke-GitHubAPI.ps1

```diff
@@ -21,11 +21,11 @@ function SplitParameterArray($members) {
     }
 }
 
-function Set-GitHubAPIParameters ($members,  $parameterName, $parameters, $allowEmptyMembers=$false) {
```

Comment:
@chidozieononiwu do you remember why we needed this allowEmptyMembers before?

Reply:
I don't see that flag being used rights now. But calling `Update-GitHubIssue` with empty members (labels or assignees) removes all the members on that issue.

Reply:
I'll experiment with this and see what the difference is between doing this with `[]` instead of no presence at all.

Reply:
I see. I've adjusted this to handle when `SplitParameterArray` returns null. If we want to allow empty members, we need to still leave the `labels` assigned to `[]`. If we do it that way, the update is non-destructive. 

Thanks very much for the callout @chidozieononiwu